src/components/access-type.js

```diff
@@ -476,24 +479,27 @@ AccessTypeBase.propTypes = {
   enableAccesstype: PropTypes.bool.isRequired,
   accessTypeKey: PropTypes.string.isRequired,
   accessTypeBkIntegrationId: PropTypes.string.isRequired,
+  prodHost: PropTypes.string,
+  stagingHost: PropTypes.string,
```

Comment:
is it possible to take only one host and do the logic based on it ?
something like
```
isStaging: true,
host: staging.abc.com
```
or 
```
isStaging: false
host: abc.com
```

Reply:
Yeah we can do that, but then the props cannot be optional right